packages/dev/docs/pages/releases/2023-11-8.mdx

```diff
@@ -0,0 +1,304 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we’re promoting React Aria Components from beta to RC, introducing built-in support for form validation, adding a new Toolbar component, and fixing plenty of bugs. Both React Aria and React Spectrum now have built-in support for form validation, including native HTML constraint validation, custom validation functions, realtime validation, and integration with server-side validation. Check out our forms guide to learn more. We also added Toolbar to React Aria Components, which provides an accessible grouping that manages arrow key navigation between interactive elements such as buttons, dropdown menus, and checkboxes. As always, huge thanks to everyone in our community for their feedback and contributions to make this release possible!
+date: 2023-11-08
+---
+
+# November 08, 2023 Release
+
+In this release, we’re promoting React Aria Components from beta to RC 🎉, introducing built-in support for form validation, adding a new Toolbar component, improving our css bundle size with [lightning css](https://lightningcss.dev), and fixing plenty of bugs.
+
+Both React Aria and React Spectrum now have built-in support for form validation, including native HTML constraint validation, custom validation functions, realtime validation, and integration with server-side validation. Check out the React Aria [forms guide](https://react-spectrum.adobe.com/react-aria/forms.html) as well as our [guide](https://react-spectrum.adobe.com/react-spectrum/forms.html) in React Spectrum to learn more.
```

Comment:
```suggestion
Both React Aria and React Spectrum now have built-in support for form validation, including native HTML constraint validation, custom validation functions, realtime validation, and integration with server-side validation. Check out the React Aria [forms guide](https://react-spectrum.adobe.com/react-aria/forms.html) as well as our React Spectrum [forms guide](https://react-spectrum.adobe.com/react-spectrum/forms.html) to learn more.
```

Reply:
maybe we should make the whole thing into a link, e.g. "React Aria forms guide" and "React Spectrum forms guide"?